Anlab.Mvc/Views/Shared/Layout.cshtml

```diff
@@ -115,11 +115,19 @@
     @RenderBody()
 
 </div>
-<footer>
-  <div class="footer_contents"><p class="discreet">&copy; The Regents of the University of California, Davis campus, 20017-@(DateTime.UtcNow.Date.ToString("yy")).</p>
-    <p class="discreet">Version: @VersionFinder.Version</p></div>
-
 
+<footer>
+    <div class="footer_contents">        
+        <p class="discreet">
+            &copy; The Regents of the University of California, Davis campus, 20017-@(DateTime.UtcNow.Date.ToString("yy")).
+        @if (User != null && User.IsInRole(RoleCodes.Admin) && ViewBag.OrderId != null)
```

Comment:
@jSylvestre oh i see.  Sorry to say but let's not do this, I don't want to search user roles on ever page load, even public pages.  We can put the lab details button into the header or something if we want, or don't even put a link to it and we can just know the URL.  We're the only ones that are going to use it anyway.  But it shouldn't be on the layout page

Reply:
@srkirkland We do it anyway for the menu above. I'm ok with changing it though.

Reply:
@jSylvestre  Cool yea we should just dump the link part